docs/outlook/insert-data-in-the-body.md

```diff
@@ -128,26 +116,20 @@ function write(message){
 }
 ```
 
-
 ## Insert data at the beginning of the item body
 
-
 Alternatively, you can use **prependAsync** to insert data at the beginning of the item body and disregard the current cursor location. Other than the point of insertion, **prependAsync** and **setSelectedDataAsync** behave in similar ways:
 
-
 - If you are prepending HTML data in a message body, you should first check for the type of the message body to avoid prepending HTML data to a message in text format.
```

Comment:
```suggestion
- If you're prepending HTML data in a message body, you should first check for the type of the message body to avoid prepending HTML data to a message in text format.
```